### PR TITLE
net-misc/gns3-server: bump for newly added openrc script

### PR DESCRIPTION
**Pull request by mm1ke**
Hi,

This updates the latest version of `gns3-server` to also install the newly added openrc script. (https://github.com/GNS3/gns3-server/commit/9d8ddea5776967ac33216ccdf36b8af7a9f12640)
With this script it's possible to use `gns3-server` without `gns3-gui` and use it with it's integrated web interface.

_I've checked https://wiki.gentoo.org/wiki/UID_GID_Assignment_Table and saw the next free `uid`/`gid` is `520`, which i took for the new user/group. Anything else i have todo here?_
**EDIT:** I've just changed the `uid`,`gid` to `521` since `520` is used by `firebird`..

Also, I've added `ubridge` to the groups of the `gns3` user (which is installed via `net-misc/ubrdige`). Technically it's not required, but `gns3` would be pretty useless without the group.
A home directory is required too since `gns3` needs a some place were it can save it's project files.

### CODE REVIEW

File: acct-group/gns3/metadata.xml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="person">
+		<email>mmk@levelnine.at</email>
+		<name>Michael Mair-Keimberger</name>
+	</maintainer>
+	<maintainer type="project">
+		<email>proxy-maint@gentoo.org</email>
+		<name>Proxy Maintainers</name>
+	</maintainer>
```
Comment (juippis) [inline acct-group/gns3/metadata.xml:11]:
```suggestion
	<maintainer type="person" proxied="yes">
		<email>mmk@levelnine.at</email>
		<name>Michael Mair-Keimberger</name>
	</maintainer>
	<maintainer type="project" proxied="proxy">
		<email>proxy-maint@gentoo.org</email>
		<name>Proxy Maintainers</name>
	</maintainer>
```

https://wiki.gentoo.org/wiki/Project:Proxy_Maintainers/User_Guide#Proxied_maintainer_in_metadata.xml

File: acct-user/gns3/metadata.xml [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="person">
+		<email>mmk@levelnine.at</email>
+		<name>Michael Mair-Keimberger</name>
+	</maintainer>
+	<maintainer type="project">
+		<email>proxy-maint@gentoo.org</email>
+		<name>Proxy Maintainers</name>
+	</maintainer>
```
Comment (juippis) [inline acct-user/gns3/metadata.xml:11]:
```suggestion
	<maintainer type="person" proxied="yes">
		<email>mmk@levelnine.at</email>
		<name>Michael Mair-Keimberger</name>
	</maintainer>
	<maintainer type="project" proxied="proxy">
		<email>proxy-maint@gentoo.org</email>
		<name>Proxy Maintainers</name>
	</maintainer>
```

https://wiki.gentoo.org/wiki/Project:Proxy_Maintainers/User_Guide#Proxied_maintainer_in_metadata.xml

File: acct-user/gns3/gns3-0.ebuild [hunk 3]
```diff
@@ -0,0 +1,12 @@
+# Copyright 2019-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit acct-user
+
+ACCT_USER_ID=521
```
Comment (juippis) [inline acct-user/gns3/gns3-0.ebuild:8]:
```suggestion
ACCT_USER_ID=522
```

File: acct-group/gns3/gns3-0.ebuild [hunk 4]
```diff
@@ -0,0 +1,8 @@
+# Copyright 2020-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit acct-group
+
+ACCT_GROUP_ID=521
```
Comment (juippis) [inline acct-group/gns3/gns3-0.ebuild:8]:
```suggestion
ACCT_GROUP_ID=522
```

**Review by juippis — COMMENTED** [review]
Tests don't work for me, getting tons of asserterrors. Do they pass for you? 

Also a note about the init.d script:
```
 * QA Notice: shell script appears to use non-POSIX feature(s):
 *    possible bashism in /etc/init.d/gns3server line 26 (should be VAR="${VAR}foo"):
 *        command_args+=" --log ${GNS3_SERVER_LOG}";
```

**Comment by mm1ke** [comment]
> Tests don't work for me, getting tons of attribute errors. Do they pass for you?
> 
> Also a note about the init.d script:
> 
> ```
>  * QA Notice: shell script appears to use non-POSIX feature(s):
>  *    possible bashism in /etc/init.d/gns3server line 26 (should be VAR="${VAR}foo"):
>  *        command_args+=" --log ${GNS3_SERVER_LOG}";
> ```

thanks @juippis for the review.
I've updated the obvious mistakes already. Regarding tests i do get the Attribute errors now too.. I don't know (yet) what the problem is, but the Attribute errors are new. I've tested older versions too and all of them are getting Attribute Errors now. Seems like it's somehow related with the testing environment...

Regarding the openrc script. I'll look into it, still have to investigate a bit further here :)

Regards

**Review by juippis — APPROVED** [review]
Thanks!

**Pull request closed without merging**